draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1161,23 +1161,11 @@ type:
 : A URN reference identifying the problem. To facilitate automated response
   to errors, this document defines a set of standard tokens for use in the
   `type` field, within the URN namespace of: "urn:ietf:params:trans:error:".
-  Other than the "malformed" error type defined in this section, each error
-  type is specific to the type of request. Specific errors are defined in the
-  respective sections below.
 
 detail:
-: A human-readable string describing the error which prevented the log from
-  processing the request.
-
-: In the case of a malformed request, the string SHOULD provide sufficient
-  detail for the error to be rectified.
```

Comment:
This sentence no longer seems to make much sense, because "malformed" has a much narrower meaning after PR #300.